### PR TITLE
add unittest for map report and infographic expressions

### PR DESCRIPTION
**Pull request by myarjunar**
### What does it fix?
* Ticket: #4582 
* Funded by: DFAT
* Description: Here I try to add a simple unittest for map_report and infographic expressions first. html_report and other expressions that require parameters and legend in the app will come next because I still have no idea how to do it.

### Checklist:
- [ ] Commit messages are descriptive and explain the rationale for changes
- [ ] Add to the changelog in metadata.txt if it's a new feature
- [ ] Unit test for new code added
- [ ] Request someone to review or test your PR

### CODE REVIEW

File: safe/report/test/test_expressions.py [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+        self.evaluate(disclaimer_title_header_element, expected_result)
+
+        # disclaimer_text_element
+        expected_result = (
+            u'InaSAFE has been jointly developed by the Indonesian '
+            u'Government-BNPB, the Australian Government, '
+            u'the World Bank-GFDRR and independent contributors. '
+            u'These agencies and the individual software developers of '
+            u'InaSAFE take no responsibility for the correctness of outputs '
+            u'from InaSAFE or decisions derived as a consequence.')
```
Comment (Gustry) [inline safe/report/test/test_expressions.py]: Can we take these strings from definitions?
If some not python developers come to update definitions, it doesn't break tests which are only strings.
Same for other tests

Comment (myarjunar) [reply safe/report/test/test_expressions.py]: so we don't want them to update the test as they update the definition?

Comment (Gustry) [reply safe/report/test/test_expressions.py]: Well, they are not aware of these tests until you launch them on travis. (even me)

I think it's a little bit annoying when you change only one character in a string, it doesn't make sense to check the content of the string, especially these strings. You should be able to fix a typo without breaking tests. The test should only check the logic, that it returns a string with some content, not checking the actual content of the string.

To sum up, updating strings like this shouldn't break test.

My opinion about what is a good test. My 2 cents ;-)

Comment (ismailsunni) [reply safe/report/test/test_expressions.py]: +1 for @Gustry 's comment
